frontend/src/modules/Survey/Components/StepRadio.tsx

```diff
@@ -19,31 +20,42 @@ export const StepRadio = ({
 }: StepProps) => {
   return (
     <StyledContainer>
-      <StyledQuestionText>
-        {question.question}
-      </StyledQuestionText>
-      <StyledRadioButtonsWrapper>
-        {showError ? (
-          <StyledErrorWrapper>
-            <StyledErrorIcon />
-            <StyledErrorText>Please select an item to continue</StyledErrorText>
-          </StyledErrorWrapper>
-        ) : null}
-        <StyledContainer>
-          {Object.entries(question.answers).map(([value, fullDescription]) => (
-            <RadioButton
-              currentAnswer={currentAnswer}
-              onClick={(e: React.ChangeEvent<HTMLInputElement>) =>
-                setAnswer(e.target.value)
-              }
-              value={value}
-              label={fullDescription}
-              name="RadioButtons"
-              key={value}
-            />
-          ))}
-        </StyledContainer>
-      </StyledRadioButtonsWrapper>
+      <main>
+        <StyledQuestionText>{question.question}</StyledQuestionText>
+        <StyledRadioButtonsWrapper>
+          {showError ? (
+            <StyledErrorWrapper>
+              <StyledErrorIcon />
+              <StyledErrorText>
+                Please select an item to continue
+              </StyledErrorText>
+            </StyledErrorWrapper>
+          ) : null}
```

Comment:
I guess this and the associated code can actually be removed now since the next button is simply disabled if they don't have anything selected yet